src/Components/EmployeeList.js

```diff
@@ -1,14 +1,38 @@
 import React from "react";
 import Employee from "./Employee";
 import "./EmployeeList.css";
-
+import { useState,useEffect } from "react";
 export const EmployeeList = () => {
+
+
+  const [data,setData]=useState(null)
+
+    
+  useEffect(()=>{
+fetch("https://vet-resource-api-9-2.herokuapp.com/api/employees")
+ .then(res => res.json()).then((data) =>    {
+ 
+//  console.log(data)
+
+setData(data)} )  
+  
+},[])
+// console.log(data)
   return (
     <main>
-      <h2>All Staff</h2>
+      {data && data.map((dat) =>{
+        console.log(dat.firstName)
+        return(
+          <div  key={dat.firstName}>
+
+         
       <section className="employee-list">
-        <Employee />
+        <div>{dat.name}</div>
+        {data && <Employee data={data} occupation={dat.title} postfix={dat.postfix} title={dat.prefix} last={dat.lastName} name={dat.firstName} id={dat.id} />}
```

Comment:
If you've got data && above, you don't need to check data again inside of that condition. Also, you could've just passed dat as an object to Employee, so something like <Employee employee={dat} />, which would mean you don't have to pass each key in the object as its own prop